addons/website_forum/static/src/scss/website_forum.scss

```diff
@@ -228,3 +228,36 @@ img.o_forum_avatar_big {
 .website_forum {
     margin-bottom: $spacer;
 }
+
+.forums main {
+    background: $gray-100;
+}
+.o_forum_card {
+    overflow: hidden;
```

Comment:
Why do you need `overflow: hidden` here ?

Reply:
The uploaded image hides the rounded corners of the parent element, that's why :-)

Reply:
Probably because you absolute-position them, which you will not need anymore when using embed classes on the col elements ;)

Reply:
I removed the absolute positioning on the image, but when placing the embed classes on the col elements, the image disappears. 
I placed the embed classes on the image and its parent instead, which seems to work well. It doesn't fix the rounded corners issue though since the embed classes add the absolute positioning.

Reply:
I was thinking of using the embed-reponsive class on the col and the embed-responsive-item on the card, then removing the absolute positioning of img and header, did this not work ?

Reply:
That's exactly what I did, but it didn't work :(

Reply:
Just tested and it works :wink: You just have to add `flex-grow-1` to `card-img` element so that it take full-height of the remaining space of the card (which is using ratio 16:9).

Reply:
Note: using on the image and its parent like you say you did is a solution too, in that case the 16:9 ratio will be on the image, in my case it will be on the card. I let you judge which is best. Note that using it on the image instead of the col may have the advantage of not forcing the border removal like the `embed-responsive-item` class is doing... so maybe we keep your solution.

Reply:
Thanks for checking :-) 
I'll keep the `embed-responsive-item` on the image, at least I don't lose the box-shadow that way.